proto.h

```diff
@@ -148,7 +148,7 @@ extern	double	errcheck(double, const char *);
 extern	int	isclvar(const char *);
 extern	int	is_number(const char *);
 
-extern	int	adjbuf(char **pb, int *sz, int min, int q, char **pbp, const char *what);
+extern	int	adjbuf(char **, int *, int, int, char **, const char *);
```

Comment:
I disagree w/the changes here. The declarations are more self documenting with the parameter names. Please revert.

Reply:
I am fine with that, but let me point out that those were the only two functions in the whole file that had parameter names.

Independently there are two tar files in the repo:
testdir/arnold-fixes.tar  testdir/beebe.tar        

Are those supposed to be there?

Reply:
Thanks for reverting the changes. The tar files are supposed to be there. The process of running the test suite untars them and runs the tests within.